package.json

```diff
@@ -73,8 +73,8 @@
     "react-dom": "17.0.2",
     "react-dropzone": "^11.3.4",
     "react-helmet": "6.1.0",
-    "react-hook-form": "^7.11.1",
     "react-i18next": "^11.11.3",
+    "react-hook-form": "^7.12.0",
```

Comment:
Can we remove `react-i18next`? There shouldn't be any import of it anymore, right? Which means we can delete the dependency.

Reply:
Nein `gatsby-plugin-react-i18next` basiert darauf. Löschen konnte ich aber `gatsby-plugin-18n`